src/components/cart/cart-page/CartItem.js

```diff
@@ -40,10 +40,10 @@ const CartItem = ({ item, handleRemoveProductClick, setCart }) => {
                 </span>
             </th>
             <td className="wd-cart-element">
-                <img
-                    width="64"
+                <Image
+                    width={64}
+                    height={64}
                     src={item.image.sourceUrl}
-                    srcSet={item.image.srcSet}
                     alt={item.image.title}
```

Comment:
Can we pull altTxt from the data and fallback to image title?
Also add optional chaining